.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,21 +31,40 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] Visual Studio
 - [ ] Eclipse IDE
 - [ ] VSCode
+- [ ] When compiling the compliler flag was used ""enable /Wall -Wformat-security -Werror=format-security"
+
+**_Delete this line and everything below if this is not a PR for a new sample_**
+
+**_Delete this line and all above it if this PR is for a new Sample_**
+# Adding a New samples
+##Introduction
 
-## Checklist for New samples:
+The full instructions on submitting a new sample is located in the [wiki](https://github.com/oneapi-src/oneAPI-samples/wiki/Contributing-a-New-Sample) if you have any questions about the steps for submitting a new code sample
 
+## Checklist
+Administrative
 - [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
```

Comment:
I've always found this one to be odd.  Generally people creating PRs don't have the power to select/add reviewers.  I would suggest leaving this out?

Reply:
Not really adding it, so much as inserting a name. I think the wiki article says that and I will remove it